src/main/java/uk/gov/hmcts/reform/pip/account/management/controllers/MediaLegalApplicationController.java

```diff
@@ -0,0 +1,100 @@
+package uk.gov.hmcts.reform.pip.account.management.controllers;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestPart;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.multipart.MultipartFile;
+import uk.gov.hmcts.reform.pip.account.management.model.MediaAndLegalApplication;
+import uk.gov.hmcts.reform.pip.account.management.model.MediaAndLegalApplicationDto;
+import uk.gov.hmcts.reform.pip.account.management.model.MediaLegalApplicationStatus;
+import uk.gov.hmcts.reform.pip.account.management.service.MediaLegalApplicationService;
+
+import java.util.List;
+import java.util.UUID;
+
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+import static org.springframework.http.MediaType.MULTIPART_FORM_DATA_VALUE;
+
+@RestController
+@Api(tags = "Account Management - API for managing media & orphaned legal professional applications")
+@RequestMapping("/application")
+public class MediaLegalApplicationController {
```

Comment:
We'll want to lock this down, probably to Admins only via the IsAdmin annotation at class level

Reply:
Have not made it class wide as think the create endpoint needs to be accessible by non admin/logged in users (based off what the UI is like), happy to make it class wide if I'm wrong :)

Reply:
i think we'll still need it to be admin, the term admin here isnt related to the user role of admin, its related to the azure access rights i believe, is that right @ChrisS1512 ? so we need it marked as admin so that only B2C admins can interact with this api, the FE being one of them

Reply:
@kurt-uk yeah that's right. Marking as resolved as solved in latest commits